src/test/kotlin/org/rust/ide/annotator/RsMacroExpansionHighlightingPassTest.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator
+
+import org.intellij.lang.annotations.Language
+import org.rust.ExpandMacros
+
+// More tests are located in `RsHighlightingAnnotatorTest` (most of those tests are executed
+// in both plain and macro context)
+@ExpandMacros
+class RsMacroExpansionHighlightingPassTest : RsAnnotationTestBase() {
+
+    fun `test attributes inside macro call`() = checkHighlighting("""
```

Comment:
Maybe also add test when some code is disabled with cfg attribute to ensure that we don't forget about `RsCfgDisabledCodeAnnotator`?